### PR TITLE
bugfix/created-at-update

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
## PR Overview
**This PR will address the following Issue/Feature:** [dbt_hubspot_source #117](https://github.com/fivetran/dbt_hubspot_source/issues/117)

**This PR will result in the following new package version:** `v0.14.0`

Since this is changing the name of a field in the end model this will be breaking.

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

### 🚨 Breaking Changes 🚨
- The `created_at` field in the `hubspot__contacts` model has been renamed to `created_date` to be consistent with the source data and avoid duplicate columns when persisting all `property_*` fields at the source package.

### Under the Hood
- Updates to the seed files and seed file configurations for the package integration tests to ensure updates are properly tested.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [X] dbt compile
- [X] dbt run –full-refresh
- [X] dbt run
- [X] dbt test
- [X] dbt run –vars (All vars in the buildkite job)

Before marking this PR as "ready for review" the following have been applied:
- [X] The appropriate issue has been linked and tagged
- [X] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [X] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [X] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [X] You have provided details below around the validation steps performed to gain confidence in these changes.

I was able to confirm that the field changes in the end model. Please see the source package for further validation checks.

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [X] indexed within dbt_project.yml
    - [X] indexed within integration_tests/dbt_project.yml
- [X] CHANGELOG has individual entries for each respective change in this PR
    
- [X] README updates have been applied (if applicable)
    
- [n/a] DECISIONLOG updates have been updated (if applicable)
- [X] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [X] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
📆

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# dbt_hubspot v0.14.0
+
+## 🚨 Breaking Changes 🚨
+- The `created_at` field in the `hubspot__contacts` model has been renamed to `created_date` to be consistent with the source data and avoid duplicate columns when persisting all `property_*` fields at the source package.
```
Comment (fivetran-jamie) [inline CHANGELOG.md:4]: could be good to link the source PR

File: README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ Include the following hubspot package version in your `packages.yml` file:
 ```yaml
 packages:
   - package: fivetran/hubspot
-    version: [">=0.13.0", "<0.14.0"] # we recommend using ranges to capture non-breaking changes automatically
+    version: [">=0.14.0", "<0.15.0"] # we recommend using ranges to capture non-breaking changes automatically
```
Comment (fivetran-jamie) [inline README.md:66]: also need to update the source package version in the dependency matrix

**Review by fivetran-jamie — APPROVED** [review]
looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:4]: Thanks for catching! Updated

File: README.md (hunk 2 above)
Comment (fivetran-joemarkiewicz) [reply README.md:66]: Yup looks like i missed this 🤦. Updated!

**Comment by fivetran-joemarkiewicz** [comment]
@fivetran-jamie thanks for reviewing! I made some updates to my branch to account for your review notes and also included some new variables (also introduced in my source PR changes) that allow users to disable the history models if they wish. Would you be able to give another glance to make sure all looks good following my updates?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,10 +1,20 @@
 # dbt_hubspot v0.14.0
 
 ## 🚨 Breaking Changes 🚨
-- The `created_at` field in the `hubspot__contacts` model has been renamed to `created_date` to be consistent with the source data and avoid duplicate columns when persisting all `property_*` fields at the source package.
+- Within the source package the `created_at` and `closed_at` fields in the below mentioned staging models have been renamed to `created_date` and `closed_date` respectively to be consistent with the source data. Additionally, this will ensure there are no duplicate column errors when passing through all `property_*` columns, which could potentially conflict with `property_created_at` or `property_closed_at`. ([PR #119](https://github.com/fivetran/dbt_hubspot_source/pull/119))
+  - `stg_hubspot__company`
+  - `stg_hubspot__contact`
+  - `stg_hubspot__deal`
+  - `stg_hubspot__ticket`
```
Comment (fivetran-jamie) [inline CHANGELOG.md:8]: should we note which transform end models are therefore affected?

**Review by fivetran-jamie — APPROVED** [review]
looks great! approved with one changelog comment

File: CHANGELOG.md (hunk 3 above)
Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md]: @fivetran-jamie I agree, I just added a sub bullet to show the downstream impacts. Surprisingly it was very straightforward in only impacting the relevant end models.

**Pull request merged**